<details><summary>Failed to submit graph for 55ffb20313308a53b4f64b77105b369a6ab6da2a
</summary>Message:
Failed to run command: 'pbp generate from-feedstock --output /tmp/tmp5c5qu7vc/scikit-image-feedstock.yaml --custom-metadata-file /tmp/tmp5c5qu7vc/custom_metadata.yaml --release-name scikit-image-feedstock-25-55ffb20313308a53b4f64b77105b369a6ab6da2a scikit-image-feedstock/recipe', with error: 'INFO:root:Found feedstock scikit-image-feedstock at /tmp/tmp5c5qu7vc/scikit-image-feedstock/recipe
INFO:root:Build/Test graph from scikit-image-feedstock
Created by sbx_user1051 on 2025-11-28 16:17:56
Command line: /opt/conda/envs/webhook-listener/bin/pbp generate from-feedstock --output /tmp/tmp5c5qu7vc/scikit-image-feedstock.yaml --custom-metadata-file /tmp/tmp5c5qu7vc/custom_metadata.yaml --release-name scikit-image-feedstock-25-55ffb20313308a53b4f64b77105b369a6ab6da2a scikit-image-feedstock/recipe

INFO:root:Found abs.yaml in /tmp/tmp5c5qu7vc/scikit-image-feedstock
INFO:pbpcli.utils.common:Setting env var: ANACONDA_ROCKET_ENABLE_PY314 = True.
INFO:root:Creating graph from feedstock at: scikit-image-feedstock
INFO:pbpcli.utils.common:Setting env var: CONDA_SUBDIR = linux-64.
INFO:pbpcli.utils.expand_variants:Elapsed time for get_package_variants: 0.09 seconds
INFO:pbpcli.utils.expand_variants:Matrix vars: 3 unchanging vars:(396), changing vars (3): ['blas_impl', 'numpy', 'python']
INFO:pbpcli.utils.expand_variants:Elapsed time for preparation for render: 0.00 seconds
Problematic recipe:



package:
  name: scikit-image
  version: 0.25.2

source:
  url: https://github.com/scikit-image/scikit-image/archive/refs/tags/v0.25.2.tar.gz
  sha256: ca7cf6861179dc60f56fdd4e7f1df88ce44984472529f66d3ff472b7e75e67a1

build:
  number: 1
  entry_points:
    - skivi = skimage.scripts.skivi:main

requirements:
  build:

    - gcc_linux-64 14.3.0.*
    - gxx_linux-64 14.3.0.*
    - pkg-config
  host:
    - python
    - cython >=3.0.8
    - lazy_loader >=0.4
    - meson-python >=0.16
    - ninja-base 1.12.1
    - numpy 2.1
    - packaging
    - pip
    - pythran >=0.16
    - setuptools >=68
  run:
    - python
    - numpy >=1.24,<3
    - scipy >=1.11.4
    - networkx >=3.0
    - pillow >=10.1
    - imageio >=2.33,!=2.35.0
    - tifffile >=2022.8.12
    - packaging >=21
    - lazy_loader >=0.4
    - _openmp_mutex
  run_constrained:
    - matplotlib-base >=3.7
    - pooch >=1.6.0
    - pyamg >=5.2
    - astropy >=5.0
    - pywavelets >=1.6
    - scikit-learn >=1.2
    - cloudpickle >=0.2.1
    - dask-core >=2021.1.0,!=2024.8.0





































test_png_round_trip
test:
  imports:
    - skimage
  source_files:
    - pyproject.toml
  requires:
    - pip
    - pooch >=1.6.0
    - asv
    - matplotlib-base >=3.7
    - pywavelets >=1.6
    - scikit-learn >=1.2

    - numpydoc >=1.7
    - astropy >=5.0
    - pytest >=8
    - pytest-localserver
    - dask-core >=2021.1.0
  commands:
    - pip check
    - python -c "import skimage; print(skimage.__version__)"
    - export OPENBLAS_NUM_THREADS=1
    - export OMP_NUM_THREADS=1
    - export MPLBACKEND=Agg
    - SKIMAGE_TEST_STRICT_WARNINGS=0 pytest --verbose --pyargs skimage -k "not (_not_a_real_test or test_analytical_moments_calculation or test_ellipse_parameter_stability or test_imsave_roundtrip or test_all_mono or test_thresholds_dask_compatibility or test_2d_bf or test_2d_cg or test_2d_cg_mg or test_2d_cg_j or test_reproducibility or test_png_round_trip)"

about:
  home: https://scikit-image.org
  license: BSD-3-Clause AND MIT AND BSD-2-Clause
  license_family: Other
  license_file: LICENSE.txt
  summary: Image processing in Python.
  description: |
    scikit-image is a collection of algorithms for image processing.
    It is available free of charge and free of restriction.
    We pride ourselves on high-quality, peer-reviewed code,
    written by an active community of volunteers.
  dev_url: https://github.com/scikit-image/scikit-image
  doc_url: https://scikit-image.org/docs/stable

extra:
  recipe-maintainers:
    - jakirkham
    - jni
    - msarahan
    - ocefpaf
    - soupault
    - hmaarrfk
    - emmanuelle

INFO:pbpcli.utils.common:Unsetting env var: CONDA_SUBDIR
INFO:pbpcli.utils.common:Unsetting env var: ANACONDA_ROCKET_ENABLE_PY314
Traceback (most recent call last):
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 418, in yamlize
    return yaml.load(data, Loader=StringifyNumbersLoader)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "yaml/_yaml.pyx", line 674, in yaml._yaml.CParser.get_single_node
  File "yaml/_yaml.pyx", line 688, in yaml._yaml.CParser._compose_document
  File "yaml/_yaml.pyx", line 732, in yaml._yaml.CParser._compose_node
  File "yaml/_yaml.pyx", line 846, in yaml._yaml.CParser._compose_mapping_node
  File "yaml/_yaml.pyx", line 732, in yaml._yaml.CParser._compose_node
  File "yaml/_yaml.pyx", line 846, in yaml._yaml.CParser._compose_mapping_node
  File "yaml/_yaml.pyx", line 730, in yaml._yaml.CParser._compose_node
  File "yaml/_yaml.pyx", line 809, in yaml._yaml.CParser._compose_sequence_node
  File "yaml/_yaml.pyx", line 861, in yaml._yaml.CParser._parse_next_event
yaml.scanner.ScannerError: while scanning a simple key
  in "<unicode string>", line 91, column 1
could not find expected ':'
  in "<unicode string>", line 92, column 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/webhook-listener/bin/pbp", line 10, in <module>
    sys.exit(cli())
             ~~~^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1442, in __call__
    return self.main(*args, **kwargs)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1363, in main
    rv = self.invoke(ctx)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1226, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 794, in invoke
    return callback(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 413, in wrapper
    return func(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 624, in from_feedstock
    build_configs, test_configs = expand_build_matrix(feedstock.recipe_dir_path, src_channels)
                                  ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 79, in expand_build_matrix
    _targets, build_configs, test_configs = variants_for_feedstock_and_platform(
                                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        recipe_dir_path, platform, src_channels
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/utils/expand_variants.py", line 544, in variants_for_feedstock_and_platform
    metadata_tuples = api.render(
        parsed.recipe,
    ...<2 lines>...
        variants=None,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/api.py", line 60, in render
    metadata_tuples = render_recipe(
        recipe_path,
    ...<4 lines>...
        permit_unsatisfiable_variants=permit_unsatisfiable_variants,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 1048, in render_recipe
    m = MetaData(str(recipe), config=config)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 1250, in __init__
    self.parse_again(permit_undefined_jinja=True, allow_no_other_outputs=True)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 1338, in parse_again
    self.meta = parse(
                ~~~~~^
        self._get_contents(
        ^^^^^^^^^^^^^^^^^^^
    ...<5 lines>...
        path=self.meta_path,
        ^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 581, in parse
    res = yamlize(data)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 422, in yamlize
    raise UnableToParse(original=e)
          ~~~~~~~~~~~~~^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/exceptions.py", line 22, in __init__
    super().__init__(*args, **kwargs)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
TypeError: CondaError.__init__() missing 1 required positional argument: 'message'
'
aws lambda request id: 77a53efd-9cbd-5efb-a3d2-5f5e51de57db
[View in Grafana](https://grafana.anacondaconnect.com/explore?schemaVersion=1&panes=%7B%2286g%22:%7B%22datasource%22:%22loki-central%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7B__aws_cloudwatch_log_group%3D%5C%22%2Faws%2Flambda%2Fpbp-graph-submitter-prod%5C%22%7D%20%7C%20json%20%7C%20requestId%20%3D%20%6077a53efd-9cbd-5efb-a3d2-5f5e51de57db%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki-central%22%7D,%22editorMode%22:%22builder%22,%22direction%22:%22backward%22%7D%5D,%22range%22:%7B%22from%22:%22now-3h%22,%22to%22:%22now%22%7D,%22panelsState%22:%7B%22logs%22:%7B%22columns%22:%7B%220%22:%22Time%22,%221%22:%22Line%22%7D,%22visualisationType%22:%22table%22,%22labelFieldName%22:%22labels%22%7D%7D%7D%7D&orgId=1)</details>

--- a/abs.yaml
+++ b/abs.yaml
@@ -1,5 +1,3 @@
-pbp_autopublish: false
-
 build_env_vars:
   ANACONDA_ROCKET_ENABLE_PY314: yes
 

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -95,6 +95,9 @@ requirements:
 #  DeprecationWarning: Saving I mode images as PNG is deprecated and will be removed in Pillow 13 (2026-10-15)
 {% set tests_to_skip = tests_to_skip + " or test_png_round_trip" %}
 
+# AssertionError: Arrays are not almost equal to 4 decimals
+{% set tests_to_skip = tests_to_skip + " or test_polynomial_weighted_estimation" %}
+
 test:
   imports:
     - skimage

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -95,7 +95,6 @@ requirements:
 #  DeprecationWarning: Saving I mode images as PNG is deprecated and will be removed in Pillow 13 (2026-10-15)
 {% set tests_to_skip = tests_to_skip + " or test_png_round_trip" %}
 
-test_png_round_trip 
 test:
   imports:
     - skimage

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -20,6 +20,7 @@ requirements:
     # pythran code needs clang-cl on windows
     - clang  # [win]
     - lld  # [win]
+    - {{ stdlib('c') }}
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
     - pkg-config

